daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -476,6 +476,8 @@
           <xs:enumeration value="unsupportedAttributeFinalDefault" />
           <xs:enumeration value="unsupportedAttributeFormDefault" />
           <xs:enumeration value="unsupportedAttributeSchemaLocation" />
+          <xs:enumeration value="discouragedDiscriminatorPlacement" />
+          <xs:enumeration value="discouragedAssertPlacement" />
```

Comment:
Can you sort these so they are at the top of the list? Makes it easier to quickly scan for warnings.

Reply:
All addressed.